content/PGP/YubiKey 5.2.3 Enhancements to OpenPGP 3.4.adoc

```diff
@@ -0,0 +1,168 @@
+== Introduction
+
+With the release of the YubiKey 5Ci device with firmware 5.2.3, Yubico offers support for the latest OpenPGP Smart Card 3.4 functionality, offering advancements in OpenPGP functionality. These enhancements allow users an expanded encryption algorithm set beyond RSA for OpenPGP operations, utilize separate x.509 cardholder certificates alongside the existing OpenPGP certificates for authentication, signature and encryption/decipher, bring attestation functionality to OpenPGP keys and certificates generated on a YubiKey and a number of quality of life improvements. Further, security has been improved as the YubiKey with firmware 5.2.3 and above supports Key Derived Format PINs, allowing the PIN to never be exposed over an insecure channel.
+
+This document is a high level review of the OpenPGP feature enhancements including in firmware 5.2.3 and above. The full specifications of what has been changed can be found in the OpenPGP Smart Card v3.4 specifications, here: https://gnupg.org/ftp/specs/OpenPGP-smart-card-application-3.4.pdf
+
+=== Functional Feature Enhancements
+YubiKey firmware 5.2.3 and OpenPGP 3.4 offers a set of new options to users, namely new support for cryptographic algorithms beyond RSA and the Yubico Attestation feature for verifying keys generated on a YubiKey device.
+
+=== Elliptic Curves
+Support for Elliptic Curve Cryptographic Algorithms have been added to the YubiKey 5.2.3 and above firmware. These curves can be used for Signature, Authentication and Decipher keys. The full list of curves supported by OpenPGP 3.4 can be found in section 4.4.3.10 of the https://www.google.com/url?q=https://gnupg.org/ftp/specs/OpenPGP-smart-card-application-3.4.pdf&sa=D&ust=1566260477095000[OpenPGP Smart Card 3.4] spec.

```

Comment:
I think we can remove google from the URL

Reply:
I think I would remove this sentence entirely.  Not sure that it adds much, and we picked the curves based on GPG support more than what the spec says.  The spec doesn't mention curve25519, for instance.